modules/creating-oci-compatible-connection-types-api.adoc

```diff
@@ -0,0 +1,147 @@
+:_module-type: PROCEDURE
+
+[id="creating-oci-compatible-connection-type-api_{context}"]
+
+= Creating an OCI-compatible connection type using the connections API
+
+[role='_abstract']
+
+In {productname-short}, you can create an OCI-compatible connection type by using the connections API. In the following procedure, you define a Kubernetes Secret for storing credentials to an OCI-compatible container registry (like Quay.io or a private registry). This allows applications to authenticate and pull container images.
```

Comment:
```suggestion
In {productname-short}, you can create a URI-compatible connection type by using the connections API. In the following procedure, you define a Kubernetes Secret for storing credentials to an OCI-compatible container registry (like Quay.io or a private registry). This allows applications to authenticate and pull container images.
```

Reply:
Just for my own clarity: Do I still need to update the sentence to "URI" if the procedure is about OCI?